doc-content/drools-docs/src/main/asciidoc/DecisionEngine/execution-control-con.adoc

```diff
@@ -17,7 +17,7 @@ ifdef::DM,PAM[]
 image::HybridReasoning/Two_Phase_enterprise.png[align="center"]
 endif::[]
 
-When multiple rules exist on the agenda, the execution of one rule may cause another rule to be removed from the agenda. To avoid this, you can define how and when rules are executed in the {DECISION_ENGINE}. Some common methods for defining rule execution order are by using rule salience, agenda groups, activation groups, or rule units for DRL rule sets.
+When multiple rules exist on the agenda, the execution of one rule may cause another rule to be removed from the agenda. To avoid this, you can define how and when rules are executed in the {DECISION_ENGINE}. Some common methods for defining rule execution order are by using rule salience, agenda groups, or activation groups.
```

Comment:
```suggestion
When multiple rules exist on the agenda, the execution of one rule may remove another rule from the agenda. To avoid this, you can define how and when the {DECISION_ENGINE} executes rules. Common methods for defining rule execution order are using rule salience, agenda groups, or activation groups.
```